crates/typst-library/src/math/accent.rs

```diff
@@ -240,9 +240,10 @@ fn create_accent_param_info() -> Vec<ParamInfo> {
 cast! {
     Accent,
     self => self.0.into_value(),
-    v: char => Self::new(v),
```

Comment:
I hadn't spotted this removal during earlier review, but this actually breaks string accents such `$ accent(.., "↔") $`.